tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/RecordSession.cs

```diff
@@ -102,6 +95,19 @@ public RecordEntry Lookup(RecordEntry requestEntry, RecordMatcher matcher, IEnum
             {
                 sanitizer.Sanitize(requestEntry);
             }
+            return LookUpSanitizedRecord(requestEntry, matcher, remove);
+        }
+
+        private RecordEntry LookUpSanitizedRecord(RecordEntry requestEntry, RecordMatcher matcher, bool remove)
+        {
+            try
+            {
+                // normalize request body with STJ using relaxed escaping to match behavior when Deserializing from session files
+                RecordEntry.NormalizeJsonBody(requestEntry.Request, requestEntry.Request.Body);
+            }
+            catch (JsonException)
+            {
+            }
```

Comment:
Nit: we explicitly catch `JsonException` in `NormalizeJsonBody`. It's not possible that we actually get this exception here. Perhaps swap to `Exception` and throw an `HttpException` if we ever seen it?

Reply:
Ah good catch - will just remove it from here.